ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -1055,6 +1023,44 @@ void WriteTexture(MaterialExtract.Channel channel, string gltfName, int index, i
             return material;
         }
 
+        private Image LinkAndStoreImage(MaterialExtract.Channel channel, SkiaSharp.SKBitmap bitmap, ModelRoot model, string fileName)
+        {
+            var existingImage = model.LogicalImages.SingleOrDefault(i => i.Name == fileName);
+            if (existingImage is not null)
+            {
+                return existingImage;
```

Comment:
It seems like such a cache check should be done way earlier, before it even has a bitmap?

Reply:
Fixed, bitmap is generated after this check. For both this and the orm texture.